### DISCUSSION TITLE
query xml not recognized?

### DISCUSSION THREAD

**Discussion by Jiehong** [nushell/nushell#10663] [Q&A]
Hello,

I checked the doc, and I've found https://www.nushell.sh/commands/docs/query_xml.html#frontmatter-title-for-filters.

So, I just tried, but nushell is not even recognizing the command:

```
$ open pom.xml | query xml content
Error: nu::shell::external_command

  × External command failed
   ╭─[entry #12:1:1]
 1 │ open pom.xml | query xml content
   ·                ──┬──
   ·                  ╰── did you mean 'url join'?
   ╰────
  help: No such file or directory (os error 2)
```

Using nushell v0.85.0.

I'm thinking that maybe https://www.nushell.sh/commands/docs/query_xml.html#frontmatter-title-for-filters comes with a different module/plugin, but it's not said in the page, so I assume it's part of the standard nushell.

What am I doing wrong?

**Comment by fdncred** [comment]
The family of `query` commands are in `nu_plugin_query`. You must build and install that, then register the plugin. You're right, our documentation should be better.
